### ISSUE TITLE
Option to hide caption on zoom

### ISSUE THREAD

**Issue by bdenham** [HiDeoo/starlight-image-zoom#7]
### Is your feature request related to a problem?

Hey @HiDeoo! I'm wondering if an option to hide the caption on zoom (or limit the caption height) at the bottom of the zoomed image. The caption bar is transparent, which helps a bit, but it does obscure the bottom parts of images with aspect ratios that are closer to 4:3 than 16:9. I already have a component that add captions for my images, so I don't need it on zoom. See screenshots.

### Images on the page:

<img width="740" alt="image" src="https://github.com/HiDeoo/starlight-image-zoom/assets/1828494/bb95e0f6-0fa0-46f6-8140-0cd44ab0eac2">

### Zoomed image (red border):

<img width="1356" alt="image" src="https://github.com/HiDeoo/starlight-image-zoom/assets/1828494/853cb47d-b513-4734-aef2-07c74cae0c1b">

An option for hiding the caption seems like a possible solution.

### Describe the solution you'd like

Option/config to hide caption bar on zoom.

### Describe alternatives you've considered

I can and have ensured that many of my images are closer to a 16:9 aspect ratio, which usually prevents the caption bar from overlapping and obsuring the image. But sometimes this isn't possible.

### Additional Context

Thanks for considering this!

**Comment by HiDeoo** [comment]
Thanks for the very detailed feedback, super appreciated :raised_hands:

This is indeed a case that totally slipped through my mind. I just released version [`0.4.0`](https://github.com/HiDeoo/starlight-image-zoom/releases/tag/v0.4.0) which now includes a new [`showCaptions`](https://starlight-image-zoom.vercel.app/configuration/#showcaptions) configuration option, still defaulting to `true` so the current behavior is preserved (and is probably the most common use case). Setting it to `false` will now prevent the caption from being displayed.

I tried to emphasize in the documentation that it should only be used in use cases like yours, where there is already another mechanism to display the caption 👍
